Fibonacci.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+public class Fibonacci {
+  public static void main(String[] args) { 
+    System.out.println("Enter an integer : "); 
+    Scanner input = new Scanner(System.in);
+    double number = input.nextDouble(); 
```

Comment:
never use double to represent integer value. 
For example:
int a = 1;
double b = a; // b can be 0.9999999999999999999 or 1.00000000000000000001
int c = b; // c will be 0 or 1 based on the moon phase